main/inc/lib/javascript/jquery-ui/jquery-ui.css

```diff
@@ -183,7 +183,7 @@
 	font-size: 100%;
```

Comment:
You can't edit third party libraries.
You should put the changes inside base.css